libparsec/crates/platform_storage/src/workspace.rs

```diff
@@ -217,6 +217,26 @@ impl WorkspaceStorage {
     }
 }
 
+// Prevent sync pattern manipulation
+impl WorkspaceStorage {
```

Comment:
For consistency, remove the dedicated `impl WorkspaceStorage` given all the other methods are already in a single impl

Reply:
done